backend/src/core/cloudstorage.py

```diff
@@ -0,0 +1,125 @@
+import io
+import logging
+import os
+from mimetypes import guess_type
+from typing import BinaryIO, Union
+
+from minio import Minio, S3Error
+from minio.datatypes import Bucket, Object
+
+logger = logging.getLogger(__name__)
+
+
+class CloudStorage:
+    def __init__(
+        self,
+        api_url: str = None,
+        access_key: str = None,
+        secret_key: str = None,
+        secure: bool = None,
+    ):
+        self.url = api_url or os.getenv("MINIO_HOST_URL")
+        self.access_key = access_key or os.getenv("MINIO_ACCESS_KEY")
+        self.secret_key = secret_key or os.getenv("MINIO_SECRET_KEY")
+        self.secure: bool = secure or os.getenv("MINIO_SECURE", False) == "True"
+        self.client = self._connect_to_cloudstorage()
```

Comment:
maybe it would be better to store some kind of config.py file in core directory, and there do operations like os.getenv, then we will export this consts from config to your class.

```
#in config.py

MINIO_HOST_URL= os.getenv('MINIO_HOST_URL')

#and then in call
from configs import MINIO_ACCESS_KEY


self.access_key = MINIO_ACCESS_KEY
```

Reply:
ok, it's good idea I forgot about config.py